client/src/pages/market/Market.css

```diff
@@ -1,8 +1,8 @@
 
-.cardcontainer{
+/* .cardcontainer{
```

Comment:
Katy comment out some of these to better our styling, but I'm leaving them so we know what we had.  Just in case something breaks.